scenes/player/Player.gd

```diff
@@ -54,11 +59,19 @@ func _ready():
 
 		synchronizer.experience_gained.connect(_on_experience_gained)
 		synchronizer.level_gained.connect(_on_level_gained)
+		
+		add_child(death_popup_instance)
+		death_popup_instance.respawn_player.connect(_on_respawn_timer_timeout)
+		death_popup_instance.hide()
+		#died_a = false #- for testing
 
 
 func _physics_process(_delta):
 	if loop_animation == "Move":
 		update_face_direction(velocity.x)
+	#if !died_a: #- for testing
```

Comment:
Make sure to remove debug code before opening a PR for review

Reply:
Done